Makefile

```diff
@@ -8,13 +8,20 @@ INSTALL ?= install
 
 PROGRAM_NAME = bitbutler
 
-install: doc
+install: doc persist
 	$(INSTALL) -D -t "${DESTDIR}${DATADIR}/${PROGRAM_NAME}" src/*.sh
 	$(INSTALL) -D man/bitbutler.1 $(DESTDIR)$(MANDIR)/man1/${PROGRAM_NAME}.1
 	mkdir -p "${DESTDIR}${BINDIR}"
 	echo -e '#!/usr/bin/env bash\nBB_VENDOR_PATH="${DESTDIR}${DATADIR}/${PROGRAM_NAME}" exec ${DESTDIR}${DATADIR}/${PROGRAM_NAME}/bitbutler.sh "$$@"' > "${DESTDIR}${BINDIR}/${PROGRAM_NAME}"
 	chmod u+rwx,g+x,o+x "${DESTDIR}${BINDIR}/${PROGRAM_NAME}"
 
+persist:
```

Comment:
This *should* probably include the `DESTDIR` as well, as otherwise it might lead to a broken state